packages/pyolite-kernel/src/worker.ts

```diff
@@ -22,6 +22,10 @@ let stderr_stream: any;
 // @ts-ignore: breaks typedoc
 let resolveInputReply: any;
 
+// TODO Import this locally
+declare var Comlink: any;
+importScripts("https://unpkg.com/comlink/dist/umd/comlink.js");
```

Comment:
yeah: `importScripts` has a a couple nast complications, and an unpkg URL is definitely a non-starter.

This has some of the cleanest comlink worker stuff i was able to muster:

https://github.com/nrbgt/janki/tree/gh-6-add-cards/js/jupyterlab-sqlite3

there's some webpack conf needed, but then it can just use `file-loader` (maybe `assets` now?) and and plain old `import`s.

Reply:
Great thanks, I wasn't sure how to tackle this, I'll have a look at what you did. Webpack is a beast.